include/HubFramework/HUBViewControllerFactory.h

```diff
@@ -58,12 +58,25 @@ NS_ASSUME_NONNULL_BEGIN
  *  for it), this method returns `nil`.
  *
  *  To be able to create a view controller without creating a feature, you can use the other view controller
- *  creation method available on this protocol.
+ *  creation methods available on this protocol.
  */
 - (nullable HUBViewController *)createViewControllerForViewURI:(NSURL *)viewURI;
 
 /**
- *  Create a view controller without a feature registration
+ *  Create a view controller without a feature registration, with implicit identifiers
+ *
+ *  @param contentOperations The content operations to use to load the content for the view controller.
+ *  @param featureTitle The title of the feature that the view controller will belong to. Used for its
+ *         default title, and also made available to contnet operations as part of `HUBFeatureInfo`.
```

Comment:
Ah, I've spelled this word a million times and still I get it wrong 😝 Nice catch!